reflow/Makefile.am

```diff
@@ -4,7 +4,7 @@ EXTRA_DIST = ReadMe.txt
 EXTRA_DIST += *.vcxproj *.vcxproj.filters
 
 SUBDIRS = .
-#SUBDIRS += dtls_wrapper/test
+SUBDIRS += dtls_wrapper/test
```

Comment:
If anything goes wrong after merge, re-commenting this line deactivates the tests, again.